citation-backoffice/src/components/forms/LinkField.js

```diff
@@ -7,7 +7,7 @@ export function toLinkInput(input, type) {
 	return {
 		__role__: 'link',
 		link: {
-			collection: link.__type__ ? link.__type__ : type,
+			collection: type === '*' ? link.__type__ : type,
```

Comment:
This change allows LinkField (same for LinksField) to take the type parameter if it's defined (i.e not the "all types" operator) and the link's type if it's not. It allows us to pass the type of a field to the function even if it's  a '*',  and this way it deletes an if condition